_apps/weekly-digest.md

```diff
@@ -0,0 +1,43 @@
+---
+# A human-friendly name of your listing
```

Comment:
Could you remove the comments in this section? Just for consistency with the other apps.